spacy_annotator/__init__.py

```diff
@@ -135,6 +138,50 @@ def __add_annotation(self, df, col_text, current_index, annotations):
                 continue
         entities = {"entities": spans}
         df.at[current_index, "annotations"] = (df[col_text][current_index], entities)
+    
+    def to_spacy(self, df, file_path: str):
```

Comment:
`filepath` requires a default.
I would put `filepath = None` as default and add logic to the method to export spacy format with a default name to current directory, for example:

```python
if file_path is None:
   file_path = os.path.join(os.getcwd(), 'annotations.spacy')
```